docs/source/index.rst

```diff
@@ -27,19 +27,37 @@ MetaSynth Documentation
 =======================
 Welcome to the `MetaSynth <https://github.com/sodascience/metasynth/>`_ documentation. 
 
-MetaSynth is a Python package for generating synthetic tabular data that statistically resembles a source dataset without exposing real values. It has two main functionalities:
+Metasynth is a Python package for generating synthetic tabular data with a focus on privacy. It is designed for owners of sensitive datasets who want to share approximations of their data for purposes like exploratory analysis and testing, without disclosing real values.
```

Comment:
Yep, but since we haven't definitively gone through with it I'm opting to stick to MetaSynth for now, that way it's all consistent throughout the docs. All instances can later be changed to follow new capitalization.